src/customer/strategies/masterpass-customer-initialize-options.ts

```diff
@@ -0,0 +1,3 @@
+export default interface MasterpassCustomerInitializeOptions {
+    container: string;
```

Comment:
Can you add a comment describing the use of this? Some thing like: 
```
    /**
     * This container is used to host the masterpass button.
     * It should be an HTML element.
     */
    container: string;
```

Reply:
Also, the name of the `container` should be something more describing... It can be `masterpassButton` or `masterpassContainer`.